doc/release-notes/5.0-release-notes.md

```diff
@@ -213,6 +213,18 @@ If this is a new installation, please see our [Installation Guide](http://guides
 
 ## Upgrade Instructions
 
+### Prerequisite: Retroactively store original file size
+
+Starting with release 4.10 the size of the saved original file (for an ingested tabular datafile) is stored in the database. We provided the following API that retrieve and permanently store the sizes for any already existing saved originals:
+
+`/api/admin/datafiles/integrity/fixmissingoriginalsizes`
+
+(see the documentation note in the Native API guide, under "[Datafile Integrity](https://guides.dataverse.org/en/5.0/api/native-api.html#datafile-integrity)").
+
+To check your installation, issue this command:
+
+   `curl https://localhost/api/admin/datafiles/integrity/fixmissingoriginalsizes`
```

Comment:
Hmm, https with no port on localhost seems weird. Maybe just http (no s) and port 8080?